### PR TITLE
III-4479 - Add organizer select

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- organizer select
- get organizers api hook
- unused-imports eslint rule

### Changed

- better typing Typeahead

---

Ticket: https://jira.uitdatabank.be/browse/III-4479

### CODE REVIEW

File: src/pages/steps/OrganizerPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+              const organizer = organizers[0];
+
+              if (isNewEntry(organizer)) {
+                return onAddNewOrganizer(organizer);
+              }
+
+              onChange(organizer);
+            }}
+            minLength={3}
+            newSelectionPrefix="Voeg nieuwe organisator toe: "
```
Comment (brampauwelyn) [inline src/pages/steps/OrganizerPicker.tsx:68]: Use translatable string

File: src/pages/steps/modals/PictureUploadModal.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ type PictureUploadModalProps = {
   onSubmitValid: (data: FormData) => Promise<void>;
 };
 
-const MAX_FILE_SIZE = 5000000;
+const MAX_FILE_SIZE = 5_000_000;
```
Comment (brampauwelyn) [inline src/pages/steps/modals/PictureUploadModal.tsx:33]: Cool 👍

**Review by brampauwelyn — APPROVED** [review]

File: .eslintrc [hunk 3]
```diff
@@ -1,9 +1,11 @@
 {
-  "plugins": ["simple-import-sort", "unicorn"],
+  "plugins": ["unused-imports", "simple-import-sort", "unicorn"],
```
Comment (geoffreydhuyvetters) [inline .eslintrc:2]: Is this with autofix? If so, didn't we enable this a while ago and removed it again because it was annoying?

**Review by geoffreydhuyvetters — APPROVED** [review]

File: .eslintrc (hunk 3 above)
Comment (simon-debruijn) [reply .eslintrc:2]: It did indeed, I've changed it to warn us instead

Comment (simon-debruijn) [reply .eslintrc]: I'm removing it again, because it still auto fixes warnings

**Pull request merged**